### ISSUE TITLE
The "Create Stub Files" command omits typed+valued class attributes

### ISSUE THREAD

**Issue by psam44** [microsoft/pylance-release#490]
## Environment data
Pylance 2020.10.0
Python 2020.9.114305

Code sample:
```python
class Foo(models.Model):
    valueOnly = 's'
    typeOnly: str
    valueAndType: str = 's'
```

The `Create Stub Files` Quick Fix generates only:
```python
class Foo(models.Model):
    valueOnly = ...
    typeOnly: str
```

Why the following property is not produced?
```python
    valueAndType: str = ...
```

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next release.

**Comment by jakebailey** [comment]
This issue has been fixed in version 2020.10.2, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/master/CHANGELOG.md#2020102-21-october-2020
